src/game/pointers/Pointers.hpp

```diff
@@ -129,6 +133,8 @@ namespace YimMenu
 	{
 		bool Init();
 		void Restore();
+		// BUMP VERSION EVERYTIME AN UPDATE IS MADE TO POINTERS
+		PointerCache Cache{1};
```

Comment:
Instead of having to manually bump it can you not write the queue size to the file each time and compare it?

Reply:
With this approach, wouldn't we have to scan for the pointer to get the size? I could be misunderstanding, but if I am interpreting it correctly, that would kind of defeat the purpose of not having to scan for it in the first place.

Reply:
@MarkEcza ?

Reply:
> With this approach, wouldn't we have to scan for the pointer to get the size? I could be misunderstanding, but if I am interpreting it correctly, that would kind of defeat the purpose of not having to scan for it in the first place.

I mean before scanning when it's added to the queue. Also you can do game version checks if you still haven't.
``` E8 ? ? ? ? 4C 8D 4C 24 ? 89 5D 68 + 1 .rip as func const char*(*)()``` will give you the build version if the sig didn't update

Reply:
Okay, so I should be saving the length of the sig too and I can check if the lengths match then update them or not. I will have to rewrite a lot with this because I won't be able to use std::pair. I mean I could just have another pair with the signature and the length. That would work too.